ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs

```diff
@@ -143,6 +143,7 @@ public override void OnDropViewInstance(ThemedReactContext reactContext, TFramew
             view.MouseEnter -= OnPointerEntered;
             view.MouseLeave -= OnPointerExited;
             _transforms.Remove(view);
+            base.OnDropViewInstance(reactContext, view);
         }
```

Comment:
Probably should be called, but doesn't technically need to be called (since we know the base implementation is empty)

Reply:
True, I iterated through a solution that required this, then backtracked... I'll remove to keep the changes to minimum.

---
In reply to: [177184095](https://github.com/Microsoft/react-native-windows/pull/1733#discussion_r177184095) [](ancestors = 177184095)